cli/flox-rust-sdk/src/models/manifest.rs

```diff
@@ -1,24 +1,269 @@
+            # The `hook.on-activate` script is run by the *bash* shell immediately upon
+            # activating an environment, and will not be invoked if Flox detects that the
+            # environment has previously been activated. Variables set by the script will
+            # be inherited by `[profile]` scripts defined below. Note that any stdout
+            # generated by the script will be redirected to stderr.
+        "#});
+
+        manifest.insert(MANIFEST_HOOK_TABLE_KEY, Item::Table(hook_table));
+
+        // `[profile]` table
+        let mut profile_table = Table::new();
+
+        match customization {
+            InitCustomization {
+                profile_common: None,
+                profile_bash: None,
+                profile_zsh: None,
+                ..
+            } => {
+                profile_table.decor_mut().set_suffix(indoc! {r#"
+
+                    # common = """
+                    #     echo "it's gettin' flox in here"
+                    # """
+                    # bash = """
+                    #     source $venv_dir/bin/activate
+                    #     alias foo="echo bar"
+                    # """
+                    # zsh = """
+                    #     source $venv_dir/bin/activate
+                    #     alias foo="echo bar"
+                    # """
+                "#});
+            },
+            _ => {
+                profile_table.insert(
+                    "common",
+                    value(formatdoc! {r#"
+                        """
+                        {}
+                        """
+                    "#, value(
+                    customization
+                        .profile_common
+                        .as_deref()
+                        .map(|profile_common| indent::indent_all_by(2, profile_common))
+                        .unwrap_or("".to_string()),
+                    )}),
+                );
+                profile_table.insert(
+                    "bash",
+                    value(formatdoc! {r#"
+                        """
+                        {}
+                        """
+                    "#, value(
+                    customization
+                        .profile_bash
+                        .as_deref()
+                        .map(|profile_bash| indent::indent_all_by(2, profile_bash))
+                        .unwrap_or("".to_string()),
+                    )}),
+                );
+                profile_table.insert(
+                    "zsh",
+                    value(formatdoc! {r#"
+                        """
+                        {}
+                        """
+                    "#, value(
+                    customization
+                        .profile_zsh
+                        .as_deref()
+                        .map(|profile_zsh| indent::indent_all_by(2, profile_zsh))
+                        .unwrap_or("".to_string()),
+                    )}),
```

Comment:
1) `toml_edit::value` twice in this is weird (kinda calls for three sets of `"""`)
2) see above comment about adding indentation and removing `"""`